README.md

```diff
@@ -146,17 +141,15 @@ Run a single test (case sensitive):
 $ npm run mocha -- test/Error.spec.js --grep 'Populates with type'
 ```
 
-If you wish, you may run tests using your Stripe *Test* API key by setting the environment variable `STRIPE_TEST_API_KEY` before running tests:
+If you wish, you may run tests using your Stripe *Test* API key by setting the
+environment variable `STRIPE_TEST_API_KEY` before running the tests:
 
 ```bash
 $ export STRIPE_TEST_API_KEY='sk_test....'
 $ npm test
 ```
 
-*Note: On Windows use `SET` instead of `export` for setting the `STRIPE_TEST_API_KEY` environment variable.*
-
-If you don't have a prefixed key (in the form `sk_test_...`) you can get one by rolling your "Test Secret Key". This can be done under your dashboard (*Account Setting -> API Keys -> Click the roll icon next to the "test secret key"*). This should give you a new prefixed key ('sk_test_..'), which will then be usable by the node mocha tests.
-
-## Author
-
-Originally by [Ask Bjørn Hansen](https://github.com/abh) (ask@develooper.com). Development was sponsored by YellowBot. Now officially maintained by Stripe.
+[connect]: https://stripe.com/connect
+[dashboard]: https://dashboard.stripe.com/account
```

Comment:
If we want we can use this direct link to API keys: https://dashboard.stripe.com/account/apikeys